src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageId>Microsoft.Azure.Management.ResourceGraph</PackageId>
     <Description>Provides Resource Graph operations for Microsoft Azure.</Description>
     <AssemblyName>Microsoft.Azure.Management.ResourceGraph</AssemblyName>
-    <Version>0.10.0-preview</Version>
+    <Version>1.0.0</Version>
```

Comment:
@chiragg4u what are the changes included in this version? Was the code regenerated for the new API version?

Reply:
There is no change in the contract; only new API version is being published as we are getting into GA of our service.
And yes the code is regenerated by the swagger update as captured @  https://github.com/Azure/azure-rest-api-specs/pull/5432

Reply:
@chiragg4u change in API version will definitely produce a diff in the generated clients, please regenerate the code,

Reply:
Regenerated code already merged; I merged with master so all the changes are present in this PR now.